proof-producer/libs/assigner/include/nil/proof-generator/assigner/trace_parser.hpp

```diff
@@ -276,17 +276,18 @@ namespace nil {
 
             // Convert storage operations
             for (const auto& pb_sop : pb_traces.storage_ops()) {
-                auto op = blueprint::bbf::storage_rw_operation(
+                throw std::logic_error("not enough data");
```

Comment:
"conversion storage operations not supported yet" + TODO in comment

Reply:
Done